app/controllers/comments_controller.rb

```diff
@@ -98,13 +99,17 @@ def create
           github_username: current_user.github_username
         }
       }
-    elsif (@comment = Comment.where(body_markdown: @comment.body_markdown,
-                                    commentable_id: @comment.commentable.id,
-                                    ancestry: @comment.ancestry)[1])
-      @comment.destroy
-      render json: { status: "comment already exists" }
+    elsif (comment = Comment.where(
```

Comment:
this assignment previously overrode `@comment` which is the comment created by the incoming params, thus hiding the validation errors we send now send back on line 111 below